azure-pipelines.yml

```diff
@@ -221,39 +221,34 @@ stages:
             ArtifactName: TestResults
           condition: failed()
 
-      - job: Publish
-        dependsOn: OtherOSes
-        pool:
-          ${{ if eq(variables._RunAsPublic, True) }}:
-            name: NetCore-Public
-            demands: ImageOverride -equals windows.vs2022.amd64.open
-          ${{ if eq(variables._RunAsInternal, True) }}:
-            name: NetCore1ESPool-Internal
-            demands: ImageOverride -equals 1es-windows-2022
-        strategy:
-          matrix:
-            Release:
-              _BuildConfig: Release
-            # ${{ if eq(variables._RunAsPublic, True) }}:
-            #   Debug:
-            #     _BuildConfig: Debug
```

Comment:
For another PR - Keeping the debug build is interesting and would avoid to have all the `Debug.Assert` failures uncaught in CI.